### ISSUE TITLE
Added version info to  help or -v 

### ISSUE THREAD

**Issue by fvant** [gr0und-s3ct0r/niet#29]
It would be great to see the version installed and determine if it is the latest by a -v option or showing it in the help text

**Comment by 4383** [comment]
Thank you for the feature proposal.
Currently you can find the version by using:
```shell
pip freeze | grep niet
```

**Comment by 4383** [comment]
Well! Now Available on [niet 1.4.0](https://pypi.org/project/niet/1.4.0/)
This feature is now available on pypi!
You can get it by updating niet:
```sh
pip install -U niet
```
Or by using the version `1.4.0`:
```sh
pip install niet==1.4.0
```
